docs/source/contents/4_draft_mapping.rst

```diff
@@ -3300,20 +3380,25 @@ Do not migrate.
 
 relatedItem[@type='host'][@displayLabel='Collection']
 -----------------------------------------------------
+
 Use Case
 ^^^^^^^^
+
 This XPath is used to indicate the resource's archival collection.
 
 Justification
 ^^^^^^^^^^^^^
+
 We use these values for search/browse facets, as well as item-level metadata display. Additionally, collection titles are shared with DPLA.
 
 XPath
 ^^^^^
+
 :code:`relatedItem[@type='host'][@displayLabel='Collection']`
```

Comment:
From a consistency standpoint, it would be nice to either use single or double quotes to enclose values in attributes (e.g. @type="host" or @type='host' consistently). I think we as a group should decided. This doesn't significantly affect the actual meaning though, so I also think we can decided to not address it. @markpbaggett & @mathewjordan what are your thoughts on this?

Reply:
It's true that I didn't think about this. Since single or double quotes are both valid in expressions (as long as they're matched or paired!), either is fine. From a consistency standpoint: I had tried (only in my work) to keep single quotes in the XPath subsections; e.g. the `originInfo` XPath section around line 1000, but then in the other subsections, used either. But I didn't apply that approach to the other XPath subsections!

If we want to make this consistent I think it would just be a matter of some time with a regex (and then some review).

Reply:
I think in the interest of getting this PR merged, let's table this for now. I'm happy to address this in the future.

Reply:
That sounds fine to me - and it makes sense: if I made this change, there'd need to be at least one more round of reviews.